Upstream-Status: Inappropriate [others]

The patch is imported from fedora project, and is not appropriate for us
to upstream it.

http://cvs.fedoraproject.org/viewcvs/devel/libatomic_ops/libatomic_ops-1.2-ppclwzfix.patch

* add fix for bad variable handling on PPC

--- libatomic_ops-1.2.orig/src/atomic_ops/sysdeps/gcc/powerpc.h	2007-05-29 19:29:52.000000000 +0200
+++ libatomic_ops-1.2/src/atomic_ops/sysdeps/gcc/powerpc.h	2007-05-29 19:30:48.000000000 +0200
@@ -72,7 +72,7 @@
   /* registers.  I always got "impossible constraint" when I	*/
   /* tried the "y" constraint.					*/
   __asm__ __volatile__ (
-    "lwz %0,%1\n"
+    "lwz%U1%X1 %0,%1\n"
     "cmpw cr7,%0,%0\n"
     "bne- cr7,1f\n"
     "1: isync\n"
